adafruit_circuitplayground/circuit_playground_base.py

```diff
@@ -717,7 +723,9 @@ def _generate_sample(self, length=100, waveform=SINE_WAVE):
         self._sample = self._audio_out(board.SPEAKER)  # pylint: disable=not-callable
         self._wave_sample = audiocore.RawSample(self._wave)
 
-    def play_tone(self, frequency, duration, waveform=SINE_WAVE):
+    def play_tone(
+        self, frequency: int, duration: float, waveform: int = SINE_WAVE
```

Comment:
Great job correctly type annotating `waveform`!  Would you mind updating the docstring's type annotation to `int` as well?